src/synthesizer/emission_models/stellar/models.py

```diff
@@ -78,26 +80,71 @@ class LineContinuumEmission(StellarEmissionModel):
     def __init__(
         self,
         grid,
-        label="linecont",
+        label="nebular_line",
         fesc=0.0,
+        fesc_ly_alpha=1.0,
         **kwargs,
     ):
         """
-        Initialise the LineContinuumEmission object.
+        Initialise the NebularLineEmission object.
 
         Args:
             grid (synthesizer.grid.Grid): The grid object to extract from.
+            fesc_ly_alpha (float): The escape fraction of Lyman-alpha.
             label (str): The label for this emission model.
             fesc (float): The escape fraction of the emission.
         """
-        StellarEmissionModel.__init__(
-            self,
-            grid=grid,
-            label=label,
-            extract="linecont",
-            fesc=fesc,
-            **kwargs,
-        )
+        if fesc_ly_alpha < 1.0:
+            # Get the lyman alpha wavelength elements and create a mask for
+            # the line (to account for spreading between bins)
```

Comment:
Comment doesn't make sense (maybe move the spreading bracket to the branches below?